point_pattern.py

```diff
@@ -27,7 +30,7 @@ def manhattan_distance(a, b):
     distance : float
                The Manhattan distance between the two points
     """
-    distance =  None # Add the algorithm to compute manhattan distance here
+    distance = sum(abs(a-b) for a,b in zip(a,b)) # Add the algorithm to compute manhattan distance here
     return distance
```

Comment:
Very nice use of `zip`, `abs`, and `sum`.  I suspect you will like the functional programming week.